frontend/src/components/broadcast/all_broadcasts/all_broadcasts.css

```diff
@@ -77,3 +77,48 @@ main {
     max-width: 100vw;
   }
 }
+
+.search {
+  background-color: rgb(255, 255, 255, 0.2);
+  border-radius: 15px;
+  margin-right: 16px;
+  position: relative;
```

Comment:
Please add `margin-left: 25px;` in here

Reply:
Done :)

Reply:
Is the color of the button ok? I can change the color if needed.

Reply:
It would be great if you can try any combination from [here](https://colors.lol/blocky) for button and it's hover effect. It would look consistnet that way :)

Reply:
![image](https://user-images.githubusercontent.com/67599130/107251463-52eb5280-6a6f-11eb-88eb-8b1034e3ae58.png)
How does this look?
I used this palette https://colors.lol/prosimian

Reply:
Hmm, looks nice 👍

Reply:
Alright, then I am done with this PR!